dts/bindings/i2c/nuvoton,npcx-i2c-port.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton, NPCX-I2C port pads node
+
+compatible: "nuvoton,npcx-i2c-port"
+
+include: [base.yaml]
+
+properties:
+    label:
+        required: true
+
+    port:
+        type: int
+        required: true
+        description: index of i2c port
+
+    controller:
+        type: phandle
+        required: true
+        description: i2c controller to handle signals from port pads
+
+    pinctrl:
```

Comment:
make this `pinctrl-0` to match convention used by other drivers/dts bindings.

Reply:
Please refer the [PR29384](https://github.com/zephyrproject-rtos/zephyr/pull/29384). Since the changes is not i2c driver only, I collect all changes in this PR. And will add related change for i2c after this PR landed.